build.sbt

```diff
@@ -1,21 +1,11 @@
 val common = Seq(
   scalaVersion := "2.12.9",
```

Comment:
can we not move to 2.13?

Reply:
I think yes we should cross build soon, not sure if all the dependencies are on 2.13 yet